res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -331,6 +351,20 @@ function sendShortMsg(control,midino,value) {
     midi.sendShortMsg(control, midino, value);
 }
 
+function parameterSoftTakeOver (group, control, value) {
```

Comment:
Why don't you use the built-in soft-takeover ?
http://www.mixxx.org/wiki/doku.php/midi_scripting#soft-takeover